scripts/upload_catalog_schema_and_organizations.py

```diff
@@ -0,0 +1,117 @@
+import argparse
+import sys
+import traceback
+from pathlib import Path
+import json
+import dotenv
+import httpx
+
+from lib.http_client import get_client
+from lib.organization import get_organization
+from lib.catalog_schema import get_schema, get_extra_fields
+from frictionless import Schema
+from lib.common import get_paths, transform_schema_field_to_payload
+from lib.format_text import format_error_message, format_success_message
+
+
+def main(directory: Path, client: httpx.Client = None) -> int:
+    code = 0
+    if client is None:
+        client = get_client()
+
+    for path in get_paths(directory):
+        organization_path = path / "organization.json"
+        organization = get_organization(organization_path)
+
+        schema_path = path / "catalog_schema.json"
+        schema = get_schema(schema_path)
+
+        payload = {"siret": organization.siret, "name": organization.name}
+
+        # Upload organization
+
+        try:
+            response = client.post("/organizations/", json=payload)
+            response.raise_for_status()
+        except (httpx.HTTPStatusError, httpx.HTTPError) as exc:
+            print(
+                format_error_message(
+                    f"ERROR: while requesting {exc.request.url!r} with {payload=}:"
+                ),
+                file=sys.stderr,
+            )
+            traceback.print_exc()
+            code = 1
+            break
+
+        if response.status_code == 201:
+            print(
+                format_success_message(
+                    f"[OK] Organization {organization.name} with {payload}"
+                )
+            )
+
+        if response.status_code > 201:
+            print(
+                format_error_message(
+                    f"ERROR: unexpected response status code: {response.status_code}"
+                ),
+                file=sys.stderr,
+            )
+            code = 1
+            break
+
+        # Upload catalog schema
+        friction_less_schema = Schema(schema_path)
+        extra_fields = get_extra_fields(friction_less_schema.field_names)
+        schema_fields = friction_less_schema.get("fields")
+        fields_payload = []
```

Comment:
(Nit) Vue son utilisation cette variable serait mieux nommée en `extra_fields_payload` (c'est ce qu'on va donner au champ `extra_fields` dans le payload).

```suggestion
        extra_fields_payload = []
```